src/app.js

```diff
@@ -1 +1,64 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { caclulateBullsAndCows } = require('./calculateBullsAndCows');
+const { generateNumber } = require('./generator');
+const { userTerminal } = require('./userTerminal');
+const { messages } = require('./constants/messages');
+
+const {
+  helloMessage,
+  requireMessage,
+  lengthError,
+  rulesMessage,
+  uniqueError,
+  winMessage,
+} = messages;
+
+const randomNumber = generateNumber();
+
+userTerminal.question(helloMessage, (answer) => {
+  if (answer === 'yes') {
```

Comment:
User can write string `'yes'` like `'Yes'`, `'yeS'`, `'YeS'` and so on. It will be a good idea to bring this string in lowercase, that we correctly understand the answer.